recipes/qarchive/all/conandata.yml

```diff
@@ -1,11 +1,19 @@
 sources:
+  "2.1.1":
+    url: "https://github.com/antony-jr/QArchive/archive/v2.1.1.tar.gz"
+    sha256: "4ed51121a5bc9b5981d2fa3927f951a6a91ccca233d6b6dc4fef55b4ca5a2d92"
   "2.0.2":
     url: "https://github.com/antony-jr/QArchive/archive/v2.0.2.tar.gz"
     sha256: "f59207c0da2d68bbbdaca79751f08018226d3b587e3f97156b3aee994db018f6"
   "2.0.1":
     url: "https://github.com/antony-jr/QArchive/archive/v2.0.1.tar.gz"
     sha256: "ee8e259795f4f991c0465c3d95cf018bea2a72e9a31f744f2344aaa43201f369"
 patches:
+  "2.1.1":
+    - patch_file: "patches/0001-cmake-conan-compatibility.patch"
+      base_path: "source_subfolder"
+    - patch_file: "patches/0003-export-symbols.patch"
```

Comment:
Please submit this patch upstream 🙏 

It would also be really nice to have https://github.com/conan-io/conan-center-index/blob/master/docs/conandata_yml_format.md#patches-fields about where this comes fro mand what it fixes

Reply:
There is no problem in making this patch upstream since I'm the maintainer of this project. This would be less work but I guess I will have to do some fixes which I have been thinking about a long time.

Reply:
It better to fix things then add patches forever 🙊  but it's fine to backcourt this patch